prime-router/metadata/schemas/covid-19.schema

```diff
@@ -53,7 +53,7 @@ elements:
     reference: Follows guidence for OBX-17 as defined in the HL7 Confluence page
     referenceUrl: https://confluence.hl7.org/display/OO/Proposed+HHS+ELR+Submission+Guidance+using+HL7+v2+Messages#ProposedHHSELRSubmissionGuidanceusingHL7v2Messages-DeviceIdentification
     hhsGuidanceField: Device ID
-    hl7Field: OBX-17-1
+    hl7OutputFields: [ OBX-17-1, OBX-17-9 ]
```

Comment:
For some receivers (NM, for example) having the value for the `device_id` in OBX-17-1 causes issues because it's too long, but they permit putting it into OBX-17-9 which allows longer values. Changed the schema here to output the value in both places.